src/support/calculateProgress.js

```diff
@@ -8,7 +8,7 @@ const calculateProgress = (goals, current) => {
     if (percentage > 100) percentage = 100;
   }
 
-  return percentage;
+  return parseFloat(percentage).toFixed(1);
```

Comment:
What does this fix, exactly? The long number shown or the fact that the number is way off?

Reply:
The problem was that the calculation returned the right value, but with 9 decimals. Apparently, SVG can't handle this, so this fixes the issue by always having the percentage at 1 decimal.